docs/docs/concepts/policies.mdx

```diff
@@ -378,41 +343,44 @@ The _Intentless Policy_ is part of Rasa's new Conversational AI with
 Language Models (CALM) approach and available starting with version `3.7.0`.
 :::
 
-The new intentless policy leverages large language models (LLMs) to complement
-existing rasa components and make it easier:
-
-- to build assistants without needing to define a lot of intent examples
-- to handle conversations where messages
-  [don't fit into intents](https://rasa.com/blog/were-a-step-closer-to-getting-rid-of-intents/)
-  and conversation context is necessary to choose a course of action.
-
-Using the `IntentlessPolicy`, a
-question-answering bot can already understanding many different ways
-that users could phrase their questions - even across a series of user messages:
-
-This only requires appropriate responses to be defined in the domain file.
-
-To eliminate hallucinations, the policy only chooses which response from
-your domain file to send. It does not generate new text.
-
-In addition, you can control the LLM by:
-
-- providing example conversations (end-to-end stories) which will be used in the prompt.
-- setting the confidence threshold to determine when the intentless policy should kick in.
-
-[This repository](https://github.com/RasaHQ/starter-pack-intentless-policy) contains a starter pack with a bot that uses the
-`IntentlessPolicy`. It's a good starting point for trying out the policy and for
-extending it.
-
-## Demo
-
-[Webinar demo](https://hubs.ly/Q01CLhyG0) showing that this policy can already
-handle some advanced linguistic phenomena out of the box.
-
-The examples in the webinar recording are also part of the end-to-end tests
-defined in the [example repository](https://github.com/RasaHQ/starter-pack-intentless-policy) in (`tests/e2e_test_stories.yml`).
-
-## Adding the Intentless Policy to your bot
+The Intentless Policy is used to send `responses` in a conversation which
+are not part of a flow. 
```

Comment:
"conversation which are not part of a flow" is confusing to understand.
The Intentless Policy is used to send `responses` to user messages that are not modelled by an existing flow?